src/app/effects/knowledge.ts

```diff
@@ -48,9 +50,15 @@ export class KnowledgeEffects {
                    this.store.dispatch(new knowledge.SearchContentAction([]));
                     return empty();
             }
-
-
-        });
+        } else {
+          this.store.dispatch(new knowledge.SearchContentAction([]));
+           return empty();
+      }
+      } else {
+        this.store.dispatch(new knowledge.SearchContentAction([]));
+         return empty();
+    }
```

Comment:
What are these else statements are for?

Reply:
I just used it to send an empty array to the action if the `response.query` is valid but the code is working fine without these else condition so I have removed it.